library/src/main/res/values/attrs.xml

```diff
@@ -27,6 +27,7 @@
     </declare-styleable>
 
     <declare-styleable name="SpeedDialView">
+        <attr name="sdReverseAnimationOnClose" format="boolean" />
```

Comment:
Please call it `sdUseReverseAnimationOnClose` to align the naming with the [FAB `useCompatPadding`](https://developer.android.com/reference/android/support/design/widget/FloatingActionButton#getusecompatpadding).

- [ ] call it `sdUseReverseAnimationOnClose`